quizproj/quizproj.cpp

```diff
@@ -0,0 +1,164 @@
+#include <stdlib.h>
+#include <string>
+#include <iostream>
+
+using namespace std;
+
+
+int main()
+{
+	int x;
+	double total;
+	
+	cout <<"This is a math quiz. It consists of ten questions." <<endl;
+	
+	cout <<"1. What is 2 + 2?" <<endl;
```

Comment:
Lets fix the spaces on the cout statements. 

cout << "This is a math quiz" << endl;